src/backend/renderer/gles2/mod.rs

```diff
@@ -537,6 +541,7 @@ impl From<Gles2Error> for SwapBuffersError {
             | x @ Gles2Error::UnexpectedSize
             | x @ Gles2Error::BlitError
             | x @ Gles2Error::CreateShaderObject
+            | x @ Gles2Error::CreateProgram
```

Comment:
Please note, that I don't know if this error should be put here or to `SwapBuffersError::ContextLost`.

Reply:
moved to `ContextLost`